.gitignore

```diff
@@ -1,3 +1,4 @@
 .env
 composer.lock
 vendor
+.idea
```

Comment:
I always think these should be in global git ignores. https://help.github.com/articles/ignoring-files/#create-a-global-gitignore